x-pack/plugins/watcher/public/sections/watch_status/components/watch_history.tsx

```diff
@@ -240,31 +241,54 @@ const WatchHistoryUI = ({ intl, watchId }: { intl: InjectedIntl; watchId: string
         aria-labelledby="indexDetailsFlyoutTitle"
       >
         <EuiFlyoutHeader>
-          <EuiTitle size="m">
-            <h2>Watch history detail</h2>
+          <EuiTitle size="s">
+            <h3>
+              <FormattedMessage
+                id="xpack.watcher.sections.watchHistory.watchHistoryDetail.title"
+                defaultMessage="Executed on {date}"
+                values={{ date: itemDetail.startTime}}
```

Comment:
getting a linting error here

Reply:
I think `startTime` needs to be added here: https://github.com/elastic/kibana/pull/35462/files#diff-e8768854560d1a6f237a10f5fe4b237dL43